modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,94 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+TBD
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage {gitops-title} from a terminal. Using this tool, users can manage ArgoCD resources, such as `Applications`, `ApplicationSets`, `AppProjects`, `User accounts`, and `GPG keys` from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature.
+====
+
+* With this update, users can configure notifications in a dedicated `NotificationsConfiguration` custom resource (CR). This configuration replaces the workflow in earlier releases of {gitops-title}. To configure notifications in earlier releases, you had to edit the `argocd-notifications-cm` ConfigMap manually. Users can now set `Triggers`, `Templates`, `Services`, and `Subscriptions` as fields in the new `NotificationsConfiguration` custom resource (CR), which is managed by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
```

Comment:
There is a note in the release note text for this [here](https://docs.google.com/document/d/1_3Hjsc3rAGvmjNo2GY_oYGqEzeyNusEOes9DNMoar9Y/edit). I think we should add something here like: 

```
Note - This update removes any configuration added to `argocd-notifications-cm` ConfigMap. We recommend users to take a backup of their configuration and update that in `default-notifications-configuration` custom resource of Kind `NotificationsConfiguration` after upgrading to the new version. Also with this update, any modifications to `argocd-notifications-cm`ConfigMap are not allowed.

```

Reply:
Thanks for providing this information @reginapizza. I have added this note after the content here.